rekognition-apis/4-custom-celebrity-recognition.ipynb

```diff
@@ -0,0 +1,1156 @@
+    "dynamodb = boto3.client('dynamodb')\n",
+    "s3 = boto3.client('s3')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# S3 bucket that contains sample images and videos\n",
+    "\n",
+    "# We are providing sample images and videos in this bucket so\n",
+    "# you do not have to manually download/upload test images and videos.\n",
+    "bucketName = \"aws-rek-immersionday-\" + awsRegion"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# DynamoDB Table and Rekognition Collection names. We will be creating these in this module.\n",
+    "ddbTableName = \"my-celebrities\"\n",
+    "collectionId = \"my-celebrities\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create temporary directory\n",
+    "# This directory is not needed to call Rekognition APIs.\n",
+    "# We will only use this directory to download images from S3 bucket and draw bounding boxes\n",
+    "\n",
+    "!mkdir m2tmp\n",
+    "tempFolder = 'm2tmp/'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DynamoDB table to store custom celebrity metadata\n",
+    "***\n",
+    "In this step we will create a DynamoDB table to store custom celebrity metadata including id, name and url. You can store additional attributes for each celebrity if needed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### List existing DynamoDB tables in your account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# List existing DynamoDB Tables\n",
+    "# Before creating DynamoDB table, let us first look at the list of existing DynamoDB tables in our account.\n",
+    "\n",
+    "listTablesResponse = dynamodb.list_tables()\n",
+    "display(listTablesResponse[\"TableNames\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create new DynamoDB Table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create new DynamoDB Table\n",
+    "        \n",
+    "createTableResponse = dynamodb.create_table(\n",
+    "    TableName=ddbTableName,\n",
+    "    KeySchema=[\n",
+    "        {\n",
+    "            'AttributeName': 'id',\n",
+    "            'KeyType': 'HASH'  #Partition key\n",
+    "        }\n",
+    "    ],\n",
+    "    AttributeDefinitions=[\n",
+    "        {\n",
+    "            'AttributeName': 'id',\n",
+    "            'AttributeType': 'S'\n",
+    "        },\n",
+    "\n",
+    "    ],\n",
+    "    ProvisionedThroughput={\n",
+    "        'ReadCapacityUnits': 10,\n",
+    "        'WriteCapacityUnits': 10\n",
+    "    }\n",
```

Comment:
I propose we switch to on-demand to save money
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Client.create_table

```suggestion
    "    BillingMode='PAY_PER_REQUEST'\n",
```